client/example_test.go

```diff
@@ -50,9 +50,12 @@ func ExampleKey_Quote() {
 		log.Fatalf("failed to create quote: %v", err)
 	}
 
+	// We don't put restriction on the hash algorithm in this example.
+	noopValidate := func(a tpm2.Algorithm) error { return nil }
```

Comment:
I'm wondering if it would work better to just have an array of allowed hash algorithms instead of passing a function handler.